eng/pipelines/templates/stages/archetype-js-release.yml

```diff
@@ -236,6 +236,11 @@ stages:
           - ${{if ne(artifact.skipPublishDevFeed, 'true')}}:
             - pwsh: |
                 $detectedPackageName=Get-ChildItem $(Pipeline.Workspace)/${{parameters.ArtifactName}}/${{artifact.name}}/*.tgz
+                if ($detectedPackageName -notmatch "-alpha")
+                {
+                  Write-Error "Found non alpha version artifact to publish as dev version. Failing publish step."
```

Comment:
Probably worth dumping out the detectedPackageName as well so we have a little more information about what we found.

Reply:
Added. Link to template release pipeline: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=977617&view=logs&j=b1e79959-24d8-5aa9-2799-72d40c3e051b&t=616ae03f-d350-5ea1-2948-454bf182d472&l=12